### PR TITLE
Import dataset after resource upload

### LINKED ISSUE #817: La MAJ "simplifiée" d'un dataset ne semble pas fonctionner
Retour d'un producteur :

> J'ai suivi la procédure mais sans supprimer le deuxième jeu de données).
En effet :
> - la procédure de mise à jour par interface simplifiée, s'achève après téléchargement du fichier sur une page blanche, sans indication de réussite ou d'échec
> - la consultation de mon compte data.gouv.fr ne permet pas de déterminer si cette opération du 24/07/2019 a eu un impact sur les jeux de données

Il faut qu'après l'envoi du fichier l'utilisateur ait la confirmation que ça a fonctionné (lui montrer la liste actualisée des ressources + un bandeau qui dit "Votre mise à jour a bien été effectuée.")

### PR DESCRIPTION
**Pull request by l-vincent-l**
So the change can be seen immediatly
fix #817

### CODE REVIEW

File: apps/transport/lib/datagouvfr/client/resources.ex [hunk 1]
```diff
@@ -3,6 +3,7 @@ defmodule Datagouvfr.Client.Resources do
   Abstraction of data.gouv.fr resource
   """
   alias Datagouvfr.Client.OAuth, as: Client
+  require Logger
```
Comment (antoine-de) [inline apps/transport/lib/datagouvfr/client/resources.ex]: is this import really required ?

File: apps/transport/lib/datagouvfr/client/oauth.ex [hunk 2]
```diff
@@ -47,6 +49,10 @@ defmodule Datagouvfr.Client.OAuth do
     url = process_url(path)
     opts = Keyword.put_new(opts, :timeout, 15_000)
     opts = Keyword.put_new(opts, :follow_redirect, true)
+    Logger.debug fn -> "Request to: #{path}" end
```
Comment (antoine-de) [inline apps/transport/lib/datagouvfr/client/oauth.ex]: do you want to keep those logs ?

File: apps/transport/lib/datagouvfr/client.ex [hunk 3]
```diff
@@ -22,6 +24,7 @@ defmodule Datagouvfr.Client do
       @spec post_process({:error, any} | {:ok, %{body: any, status_code: any}}) ::
               {:error, any} | {:ok, any}
       def post_process(response) do
+        Logger.debug fn -> "response: #{inspect(response)}" end
```
Comment (antoine-de) [inline apps/transport/lib/datagouvfr/client.ex]: do you want ot keep this log ?

Comment (l-vincent-l) [reply apps/transport/lib/datagouvfr/client.ex]: yep this is useful when you develop to know exactly what happened

File: apps/transport/lib/datagouvfr/client/oauth.ex (hunk 2 above)
Comment (l-vincent-l) [reply apps/transport/lib/datagouvfr/client/oauth.ex]: yep this is useful when you develop to know exactly what happened

File: apps/transport/lib/datagouvfr/client/resources.ex (hunk 1 above)
Comment (l-vincent-l) [reply apps/transport/lib/datagouvfr/client/resources.ex]: nop, I'll remove it

**Pull request merged**